node/tools/src/k8s.rs

```diff
@@ -121,6 +128,38 @@ pub async fn create_or_reuse_namespace(client: &Client, name: &str) -> anyhow::R
     }
 }
 
+pub async fn add_chaos_delay_for_node(client: &Client, node_name: &str) -> anyhow::Result<()> {
```

Comment:
This needs to support teardown/cleanup.

Reply:
I updated the function signature to accept the duration of the delay as a parameter. Once the delay completes, it will not remove the chaos action from the cluster but it will stop. Perhaps we could introduce another function to delete that action, which we can then call at the end of the test. What do you think about this approach?

Reply:
Yes, full cleanup need to be supported, to be called prior to the end of the test, in accordance with the test scenario, which might not rely on clock time. And in addition, for not introducing test code bugs due to unpredictable cluster state.